GPU/GPUTracking/Base/cuda/CMakeLists.txt

```diff
@@ -114,13 +114,27 @@ if(ALIGPU_BUILD_TYPE STREQUAL "O2")
     ${MODULE}
     SOURCES ${SRCS}
     PUBLIC_LINK_LIBRARIES O2::GPUTracking O2::ITStrackingCUDA
+    PRIVATE_LINK_LIBRARIES ONNXRuntime::ONNXRuntime
     PRIVATE_INCLUDE_DIRECTORIES
       ${CMAKE_SOURCE_DIR}/Detectors/Base/src
       ${CMAKE_SOURCE_DIR}/Detectors/TRD/base/src
       ${CMAKE_SOURCE_DIR}/DataFormats/Reconstruction/src
       ${CMAKE_CURRENT_SOURCE_DIR}
     TARGETVARNAME targetName)
 
+  message("Compile definitions for ONNX runtime (CUDA):")
```

Comment:
I would not put all of this here, CUDA build is only done when compiling with CUDA, so the printout will only be there if we build for CUDA, which makes no sense for the ROCm variable to be honest. Do we actually need this, since we anyway have this printout in the cmake line with -DCMAKE....

In general, I would propose to do the following:
- Remove the printouts here.
- Create a separate PR, in which:
  - you move the `include(dependencies/FindONNXRuntime.cmake)` from the main CMakeLists.txt to `dependencies/O2Dependencies.cmake`. In there, you create a one-line printout that shows "ONNXRuntime Found: 0/1 ORT_CUDA_BUILD 0/1 ...".